node/src/container_chain_spawner.rs

```diff
@@ -173,6 +177,29 @@ impl ContainerChainSpawner {
             // Update CLI params
             container_chain_cli.base.para_id = Some(container_chain_para_id.into());
 
+            // Force container chains to use warp sync
+            // If the container chain is still at genesis block, use full sync because warp sync is broken
+            let container_chain_is_at_genesis = if !orchestrator_runtime_api
```

Comment:
I would call this variable `full_sync_needed`, because `container_chain_is_at_genesis` does not accurately represent when the runtime-api is not there (the container-chain might not be at genesis here)